docs/connectors/getting-started-connectors/README.adoc

```diff
@@ -78,11 +78,16 @@ WARNING: This content is generated by running npm --prefix .build run generate:a
 END GENERATED ATTRIBUTES
 ////
 
+
+:openshift: OpenShift
+:openshift-dedicated: OpenShift Dedicated
+
```

Comment:
Do not add new attributes directly to a README.adoc file. 

Instead, add the new attributes to the attributes doc (https://github.com/redhat-developer/app-services-guides/blob/main/docs/_artifacts/document-attributes.adoc), and then regenerate the attributes in all README.adoc files as described in the "Attribute injection" section of the https://github.com/redhat-developer/app-services-guides/blob/main/CONTRIBUTING.adoc file.